deploy.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+VAULT_TOKEN=$1
+GIT_BRANCH=$2
+
+#need to get the environment from the branch name
+if [ "$GIT_BRANCH" == "develop" ]; then
+	ENVIRONMENT="dev"
+elif [ "$GIT_BRANCH" == "alpha" ]; then
+        ENVIRONMENT="alpha"
+elif [ "$GIT_BRANCH" == "staging" ]; then
+	ENVIRONMENT="staging"
+elif [ "$GIT_BRANCH" == "master" ]; then
+	ENVIRONMENT="prod"
+elif [ "$GIT_BRANCH" == "gawb3639" ]; then
+	ENVIRONMENT="dev"
```

Comment:
deploying my branch to dev for now to make sure it works - will remove this before merging

Reply:
Sorry to butt-in on a random PR, but you might think about whether your needs here are unique. If not, this might be a nice feature to build-in for the team. For example, you optionally read a `.deploy` file that contains this option and instruct devs to create one (which is in the `.gitignore`) to get this behavior. Just a thought—take with a grain of salt.

Reply:
It's a nice idea, but it's not isolated to here. If running this script from a local workstation, one can just pass in `develop` or whatever is appropriate. The need for recognizing other branches is for triggering deploys from Jenkins which also requires the branch to be listed here: https://github.com/broadinstitute/dsp-jenkins/blob/master/src/main/resources/IndieDeployConfig.conf.